feature/space-entrepreneurship-funding-sources.md

```diff
@@ -236,69 +247,75 @@ Here's the funding source breakdown:
+  * [Luxembourg Space Agency](https://space-agency.public.lu/en/funding/terms-of-participation.html) 🚀
 
 ### Challenges / Hackathons
 
+*Challenges and Hackathons can often provide a place for the seed of an idea to gain initial vetting and garner some attention. Early teams can sometimes form at these events, or be given the initial validation they need to take the next step.*
+
 * Australia
-  * [Gravity Challenge](https://gravitychallenge.space/)
+  * [Gravity Challenge](https://gravitychallenge.space/) 🚀
 * New Zealand
-  * [NZ Aerospace Challenge](https://nzaerospacechallenge.com/)
+  * [NZ Aerospace Challenge](https://nzaerospacechallenge.com/) 🚀
 * US
   * [Air Force AFWERX Challenge](https://afwerxchallenge.com/)
-  * [GEN Space](https://www.genglobal.org/gen-space)
-  * [Mars City Design Challenge](https://www.marscitydesign.com/)
-  * [Newspace Business Plan Competition](https://cscf.space/newspace-bpc)
-  * [Space Apps Challenge 2020](https://www.spaceappschallenge.org/)
+  * [GEN Space](https://www.genglobal.org/gen-space) 🚀
+  * [Mars City Design Challenge](https://www.marscitydesign.com/) 🚀
+  * [Newspace Business Plan Competition](https://cscf.space/newspace-bpc) 🚀
+  * [Space Apps Challenge 2020](https://www.spaceappschallenge.org/) 🚀
 
 ### Conferences / Pitch Events
 
+*Much like challenges and hackathons, conferences and pitch events can often provide a place for an idea to garner some attention as well as potential customers and investors.*
+
 * US
-  * [IAC Pitch sessions](https://www.iac2019.org/)
-  * [MIT New Space Age Conference](http://newspaceage.org/) 
-  * [SATELLITE Conference & Exhibition](https://www.satshow.com/)
-  * [SmallSat](https://smallsat.org/)
-  * [Space Invest Summit](https://cscf.space/space-investment-summit)
-  * [Space Symposium](https://www.spacesymposium.org/)
-  * [Spacecom](https://spacecomexpo.com/)
-  * [TC Sessions: SPACE](https://techcrunch.com/events/tc-sessions-space-2020/) 
+  * [IAC Pitch sessions](https://www.iac2019.org/) 🚀
+  * [MIT New Space Age Conference](http://newspaceage.org/) 🚀
+  * [SATELLITE Conference & Exhibition](https://www.satshow.com/) 🚀
+  * [SmallSat](https://smallsat.org/) 🚀
+  * [Space Invest Summit](https://cscf.space/space-investment-summit) 🚀
+  * [Space Symposium](https://www.spacesymposium.org/) 🚀
+  * [Spacecom](https://spacecomexpo.com/) 🚀
+  * [TC Sessions: SPACE](https://techcrunch.com/events/tc-sessions-space-2020/) 🚀
 
 <!-- End Links (do not remove me) -->
 
 ## License
 
 [![CC0](http://mirrors.creativecommons.org/presskit/buttons/88x31/svg/cc-zero.svg)](https://creativecommons.org/publicdomain/zero/1.0/)
 
-To the extent possible under law, [The Orbital Index](https://orbitalindex.com/) has waived all copyright and related or neighboring rights to this work.
+To the extent possible under law, [The Orbital Index](https://orbitalindex.com/) and its contributors have waived all copyright and related or neighboring rights to this work.
```

Comment:
Seems like we should note this as well.

Reply:
We could probably just drop the license.